src/utils/temp.test.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import { getTempFolder } from './temp';
+import * as fs from 'fs';
+
+describe('temp folder test', () => {
```

Comment:
Non-blocking test nit:

We could improve this by adding or including a test for the cleanup function

```
desc: getTempFolder function returns a folder

let tempFolderPath
before: tempFolderPath = getTempFolder()

it: that exists (expect exist)
it: that contains the correct subfolders (expect folders)


desc: cleanUpTempFolder function

it: removes the temporary folder from the local system
  getTempFolder
  expect exist
  cleanUpFolder
  expect NOT exist
```